projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -323,7 +330,26 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit, AfterViewChe
     /**
      * Desired height of the grid. If unspecificed, the grid fills the parent container.
      */
-    @Input() height: number;
+    @Input() set height(height: number) {
+        this._height = height;
+        if (this.height) {
+            this.node.nativeElement
+                .querySelector('clr-datagrid')
+                .style.setProperty('--datagrid-height', this.height + 'px');
+        } else {
+            this.node.nativeElement.querySelector('clr-datagrid').style.setProperty('--datagrid-height', 'unset');
+        }
+    }
+
+    get height(): number {
+        return this._height;
+    }
+
+    private _height: number;
+
+    @HostBinding('class.fill-parent') get fillParent(): boolean {
```

Comment:
can we name this `shouldFillParent`? fillParent looks like you're trying to do something.